scripts/ws-clone.sh

```
@@ -33,28 +38,35 @@ clone_component() {
-    git_org=$(yq '.defaults.gitOrg' "$ECOSYSTEM")
+    git_org=$(yq '.defaults.gitOrg' "$ECO")
     local repo_url="$git_org/$name.git"
 
     echo "CLONE: $name -> $target"
     git clone "$repo_url" "$target"
 }
 
 if [[ "${1:-}" == "--all" ]]; then
-    for name in $(yq '.components | keys | .[]' "$ECOSYSTEM"); do
+    # Safety check: no components declared
+    local comp_count
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)